graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -403,10 +480,23 @@ def link_schema_elements(elements, inheritance_structure):
 #              explicit value for this property. Set to None if no default is given in the schema.
 PropertyDescriptor = namedtuple('PropertyDescriptor', ('type', 'default'))
 
-# A way to describe a schema's normalized inheritance structure.
+
+# A way to describe a schema's normalized inheritance structure:
```

Comment:
Please don't modify code or comments unrelated to the current PR, especially for things like whitespace and punctuation. The benefit that introduces is not worth it relative to the cost of making the git history of the file more convoluted.